components/utilities/UNSAFE_direction/index.jsx

```diff
@@ -25,5 +25,12 @@ DIRECTIONS.RTL = 'rtl';
 // eslint-disable-next-line camelcase
 const UNSAFE_DirectionSettings = React.createContext('ltr');
 
+const makeRTL = (component) => (
+	// eslint-disable-next-line
+	<UNSAFE_DirectionSettings.Provider value="rtl">
+		<div dir="rtl">{component}</div>
+	</UNSAFE_DirectionSettings.Provider>
+);
+
 // eslint-disable-next-line camelcase
-export { UNSAFE_DirectionSettings as default, DIRECTIONS };
+export { UNSAFE_DirectionSettings as default, DIRECTIONS, makeRTL };
```

Comment:
Once #2089 and #2077 are merged I can rebase and update all duplicated `makeRtl` calls to refer to this centralized one. If you disagree and think we should just repeat it wherever needed please let me know.

Reply:
I don't think this should be in `src`. It's more of a test utility.

Reply:
Sure. I'll find a better place for it. It's also used in storybook code, other than tests.